internal/storage/image.go

```diff
@@ -861,7 +861,7 @@ func GetImageService(ctx context.Context, store storage.Store, serverConfig *con
 		imageCache:           make(map[string]imageCacheItem),
 		ctx:                  ctx,
 		config:               serverConfig,
-		regexForPinnedImages: CompileRegexpsForPinnedImages(serverConfig.PinnedImages),
+		regexForPinnedImages: CompileRegexpsForPinnedImages(append(serverConfig.PinnedImages, config.DefaultPauseImage)), // add sandbox/pause image to the list of pinned_images
```

Comment:
can you elaborate on rational here? like say how we always add the sandbox image even if not specified to protect the user and sandbox image or something more eloquent

Reply:
Addressed this concern in the latest push. Let me know your thoughts.